README.md

```diff
@@ -84,6 +84,74 @@ spec:
 When the service is up and running, it's possible to interact with the glance
 API and upload an image using the Ceph backend.
 
+## Example: Testing Glance policies
```

Comment:
cool, nice we have an example to keep testing the functions of the deployed `Glance` service.
From the previous conversations, I see the agreement is not to use `install_yamls` project to build project/users that can be used by the Galnce operator, but from a dev perspective, I think it's useful having a way to deploy the operator, see the service up & running, and try to list or even upload an image.
We can maybe provide a dev/ directory into hack as a place where we can store scripts (e.g. bash scripts) to help w/ development, but definitely something we can follow up later.
Thanks for this patch Abhishek!